### ISSUE TITLE
Convert class components to function components

### ISSUE THREAD

**Issue by somya51p** [FNNDSC/ChRIS_ui#248]
Most of the pages are written using function components but few are written using class components. It would be better if all are converted to react-hooks. 
Starting with pages like `NotFound.tsx` and `BasicConfiguration.tsx`. 
Should I work on this issue?

@PintoGideon @jennydaman Please correct me if I am missing out on something.

**Comment by mocktarissa** [comment]
Yes I am already working on that I will submit a PR as soon as I'm done.

**Comment by jennydaman** [comment]
Duplicate of https://github.com/FNNDSC/ChRIS_ui/issues/206

**Comment by somya51p** [comment]
@jennydaman  In that issue, we have converted 4 pages, I have created this issue to work on other pages.
